Adamant/Models/CoreData/Chatroom+CoreDataClass.swift

```diff
@@ -23,6 +23,11 @@ public class Chatroom: NSManagedObject {
         lastTransaction?.isUnread = false
     }
     
+    func markAsUnread() {
+        hasUnreadMessages = true
+        lastTransaction?.isUnread = true
+    }
```

Comment:
Why don't we use the exactly same algorithm as in `func markAsReaded()` but the opposite?

Reply:
Discussed it on the call